### ISSUE TITLE
breaking change: quotes around values disappeared from php.ini

### ISSUE THREAD

**Issue by jorhett** [chef-cookbooks/php#141]
you've introduced a breaking change where values placed in the PHP files no longer have quotes. This was entirely unexpected, and isn't even mentioned in your changelog.

```
-date.timezone="America/Los_Angeles"
-display_errors="Off"
-error_reporting="E_ALL & ~E_DEPRECATED & ~E_STRICT"
-expose_php="off"
-memcache.hash_strategy="standard"
-short_open_tag="on"
+date.timezone=America/Los_Angeles
+display_errors=Off
+error_reporting=E_ALL & ~E_DEPRECATED & ~E_STRICT
+expose_php=off
+memcache.hash_strategy=standard
+short_open_tag=on
```

In particular, the error_reporting doesn't handle that situation at all. This is a breaking change for most people, it needs to be clear in the documentation.

**Comment by jorhett** [comment]
The breaking change was here: https://github.com/opscode-cookbooks/php/commit/58bcea49bd0544126d571fa928c09b0d24cc4dc0

I don't object to the change, but it definitely needs to warn users about it.

**Comment by Rockstar04** [comment]
Ah, @jorhett you are right, I forgot about things like date.timezone needing quotes.

**Comment by Rockstar04** [comment]
For anyone coming here for this issue, the reference PR is #89 and the first issue in #88 

I added to the changelog here on Github, but at this point I can’t change the log that is on the Supermarket or that was sent out in the email from supermarket if you follow the cookbook there.

I had labeled this as a bug since it was breaking the use of error reporting, and it made sense to me that users could manually add quotes where needed. I didn’t even think about user defined directives since our array here is empty I didn’t have to make any changes other than merge the PR.

This was a major mistake on my part. I understand the idea of SemVer, and strive to follow it. This break in BC was definitely not intended and I am sorry for any issues this may have caused anyone.
